### PR TITLE
Groundwork for new layout

### PR DESCRIPTION
**Pull request by goodvibes**
Adds changes required for new layout
Adds supporting component
Some legibility tweaks for existing files to help with comparison

### CODE REVIEW

**Comment by ruthdas03** [comment]
The tests seem to be failing, can it be fixed please.

File: build.sbt [hunk 1]
```diff
@@ -130,7 +132,7 @@ lazy val microservice = Project(appName, file("."))
   .settings(scalaSettings: _*)
   .settings(publishingSettings: _*)
   .settings(defaultSettings(): _*)
-  .settings(scalaVersion := "2.12.13")
+  .settings(scalaVersion := "2.13.8")
```
Comment (nubz) [inline build.sbt:135]: can we check why we are updating this and that this is the version to use?

File: app/uk.gov.hmrc.helptosavestridefrontend/config/FrontendAppConfig.scala [hunk 2]
```diff
@@ -18,6 +18,7 @@ package uk.gov.hmrc.helptosavestridefrontend.config
 
 import javax.inject.{Inject, Singleton}
 import play.api.{Configuration, Environment}
+import play.twirl.api.{Html, HtmlFormat}
```
Comment (lutnos) [inline app/uk.gov.hmrc.helptosavestridefrontend/config/FrontendAppConfig.scala:21]: I very much doubt this change is needed

File: app/uk.gov.hmrc.helptosavestridefrontend/models/NSIPayload.scala [hunk 3]
```diff
@@ -62,8 +62,20 @@ object NSIPayload {
...
+      ContactDetails(
+        details.address1,
+        details.address2,
+        details.address3,
+        details.address4,
+        details.address5,
+        details.postcode,
+        Some(details.countryCode)
+      )
+    )
```
Comment (lutnos) [inline app/uk.gov.hmrc.helptosavestridefrontend/models/NSIPayload.scala:79]: I'd prefer you didn't commit this change, it is an auto-formatting change, I assume.  But why change a file who's functionality you aren't changing, it just adds to history of the file, it seems to me.

If you are sue that this is formatting by scalafmt and not going to change again, then I can see yopu could argue for including it.

Your call

**Review by lutnos — DISMISSED** [review]
Please remove the reduncdant import at least.  Config change look fine

**Review by nubz — APPROVED** [review]
LGTM - we know the layout may end up having more features such as back link and page title handling but as it's not called from any templates yet this is a good foundation

**Pull request merged**